go/zeusmaster/zeusmaster_test.go

```diff
@@ -0,0 +1,265 @@
+package zeusmaster_test
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/burke/zeus/go/config"
+	slog "github.com/burke/zeus/go/shinylog"
+	"github.com/burke/zeus/go/unixsocket"
+	"github.com/burke/zeus/go/zeusmaster"
+)
+
+var testFiles = map[string]string{
+	"zeus.json": `
```

Comment:
It is kinda gross how these files are embedded here, but I think outside pulling in some static file embedding library, that might be the best hope we have?